tools/docker/Dockerfile

```diff
@@ -9,6 +9,8 @@ RUN apt-get update  &&
     apt-get -y install wget &&                                                   \
     apt-get -y install python3 python3-dev python3-pip &&                        \
     apt-get -y install libblas-dev liblapack-dev &&                              \
+    apt-get -y install libhdf5-dev &&                                            \
+    apt-get -y install libboost-test-dev &&                                      \
```

Comment:
HighFive's cmake complains about missing `boost_unit_test_framework`
https://circleci.com/gh/STEllAR-GROUP/phylanx/1161
Let me see if we can avoid that test. Since HighFive is header-only library we don't need the test to install HighFive

Reply:
Ok, understood. If we end up needed this Boost library I'd suggest adding it to the base docker file to avoid Boost version mismatches.